scripts/check_queue_periodicity.py

```diff
@@ -193,21 +198,15 @@ def do_queue_query(url, queue_name):
         },
         "sort": [{"job.job_info.time_start": {"order": "desc"}}],
         "_source": ["job_id", "status", "job_queue", "payload_id", "payload_hash", "uuid",
-                    "job.job_info.time_queued", "job.job_info.time_start",
+                    "job.job_info.time_queued", "job.job_info.time_start", "job.job_info.time_limit",
                     "job.job_info.time_end", "error", "traceback"],
         "size": 1
     }
     logging.info("query: %s" % json.dumps(query, indent=2, sort_keys=True))
 
     # query
-    url_tmpl = "{}/job_status-current/_search"
-    r = requests.post(url_tmpl.format(url), data=json.dumps(query))
-    if r.status_code != 200:
-        logging.error("Failed to query ES. Got status code %d:\n%s" %
-                      (r.status_code, json.dumps(query, indent=2)))
-    r.raise_for_status()
-    result = r.json()
-
+    ES = elasticsearch.Elasticsearch(url)
+    result = ES.search(index="job_status-current", body=json.dumps(query))
```

Comment:
@mkarim2017 you should use this connection instead: https://github.com/hysds/hysds/blob/develop/hysds/es_util.py

so that in case we move mozart to AWS Elasticsearch, we dont have to refactor this code

example: https://github.com/hysds/hysds/blob/develop/hysds/user_rules_job.py#L16-L28

Reply:
I had to put it in. a  try except clause as not everything that is required may not be there.
we can get error: 
ModuleNotFoundError: No module named 'aws_requests_auth'ModuleNotFoundError: No module named 'aws_requests_auth'